packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -0,0 +1,167 @@
+              <Button
+                className="btn-tertiary"
+                onClick={dismissChangelog}
+                tag="a"
+                href={post.permalink}
+                data-testid="changelogReleaseNotesBtn"
+              >
+                Release notes
+              </Button>
+              {isExtension && (
+                <Button
+                  tag={isFirefoxExtension ? 'a' : undefined}
+                  href={
+                    isFirefoxExtension
+                      ? 'https://r.daily.dev/firefoxupdate'
+                      : undefined
+                  }
+                  className="bg-cabbage-40 btn-primary"
+                  data-testid="changelogExtensionBtn"
+                  onClick={async () => {
+                    if (isFirefoxExtension) {
+                      return;
+                    }
+
+                    if (isExtension) {
+                      const sendMessage =
+                        globalThis?.browser?.runtime?.sendMessage;
+
+                      if (typeof sendMessage === 'function') {
+                        const updateResponse: { status: string } =
+                          await sendMessage({
+                            type: ExtensionMessageType.RequestUpdate,
+                          });
+
+                        let toastMessage: string;
+
+                        switch (updateResponse.status) {
+                          case 'throttled':
+                            toastMessage =
+                              'There is no update available, try again later';
+                            break;
+                          case 'no_update':
+                            toastMessage =
+                              'You are already on the latest available version';
+                            break;
+                          case 'update_available':
+                            toastMessage = 'Browser extension updated';
+                            break;
+                          default:
+                            break;
+                        }
+
+                        if (toastMessage) {
+                          toast.displayToast(toastMessage);
+                        }
+                      }
+                    }
+                  }}
+                >
+                  Update extension
+                </Button>
+              )}
+            </footer>
+          </div>
+        )
+      }
+      offset={[6 * 16, 2.5 * 16]}
+      interactive
+      container={{
+        className: 'shadow',
+        paddingClassName: 'p-0',
+        roundedClassName: 'rounded-16',
+        bgClassName: 'bg-cabbage-40',
+        arrowClassName:
+          'bg-cabbage-40 bottom-24 !left-0 !h-2.5 !w-0 flex items-center justify-end before:!w-10 before:!h-px before:!transform-none',
+      }}
+      reference={elementRef}
+      arrow
+      placement="right-end"
+      visible={!!post}
+      zIndex={9}
```

Comment:
@sshanzel FYI add this so it is below settings and other modals, as far as I can see the modals are `z-index: 10`

Reply:
Great catch! 🚀